src/book/03-guides/01-main-codebase/03-camera-calibration.mdx

```diff
@@ -43,7 +22,7 @@ The calibration requires two steps, creating the `nbs` recording and using it to
 
     The serial numbers can be retrieved by first running `lsusb` and finding the bus and device numbers for the FLIR/Point Grey cameras. Then run `lsusb -s <bus number>:<device number> -v` to find the serial numbers.
 
-    Next you need to figure out which serial number is left and which is right. Change the serial number for one camera to an incorrect number (this ensures it's not found when running cameratest). Enable compressed images in `NUsight.yaml`. Run `./cameratest` and test which camera is working by covering one camera and seeing the resulting image in the NUsight vision tab. Alternatively, you could unplug one camera rather than change the serial number. If the serial number is on the wrong camera configuration file, switch them.
+    Next you need to figure out which serial number is left and which is right. Change the serial number for one camera to an incorrect number (this ensures it's not found when running cameratest). Enable compressed images in `NetworkForwarder.yaml`. Run `./cameratest` and test which camera is working by covering one camera and seeing the resulting image in the NUsight vision tab. Alternatively, you could unplug one camera rather than change the serial number. If the serial number is on the wrong camera configuration file, switch them.
```

Comment:
```suggestion
    Next you need to figure out which serial number is left and which is right. Change the serial number for one camera to an incorrect number (this ensures it's not found when running cameratest). Enable `message.output.CompressedImage` in `NetworkForwarder.yaml`. Run `./cameratest` and test which camera is working by covering one camera and seeing the resulting image in the NUsight vision tab. Alternatively, you could unplug one camera rather than change the serial number. If the serial number is on the wrong camera configuration file, switch them.
```

Reply:
Not entirely relevant but sure.

Reply:
Yeah I was here and though why not.